lib/axon.ex

```diff
@@ -3495,8 +3487,36 @@ defmodule Axon do
   or backend.
   """
   @doc type: :compilation
-  def compile(model, opts \\ []) when is_list(opts) do
-    {Axon.Compiler.compile_init(model, opts), Axon.Compiler.compile_predict(model, opts)}
+  def build(model, opts \\ []) when is_list(opts) do
+    {init_fn, predict_fn} = Axon.Compiler.build(model, opts)
+    opts = [on_conflict: :reuse] ++ opts
+    # TODO: Wrap in jit_apply
```

Comment:
Yes, the `on_conflict: :reuse` assumes you are using Nx v0.3, which means Nx.Defn.jit/3 should be printing deprecation warnings. :)